app/views/streams/_header.html.erb

```diff
@@ -2,7 +2,10 @@
   <h2 class="font-weight-normal mb-0">Stream: <%= stream.display_name %></h2>
 
   <div class="p-2">
-    <%= content_tag :span, 'Default', class: 'badge bg-info me-3' if stream.default? %>
+    <%= default_stream_status_badge(stream) if stream.default? %>
```

Comment:
This isn't displaying all the start/end times for the currently default stream if it was the default more than once. I have 2 streams with the following histories:

```
[[#<DefaultStreamHistory:0x00007f1453af2878                  
   id: 6,                                                    
   stream_id: 6,                                             
   start_time: Wed, 11 May 2022 19:06:41.991248000 UTC +00:00,
   end_time: Thu, 12 May 2022 20:15:20.589156000 UTC +00:00>,
  #<DefaultStreamHistory:0x00007f1453af2670
   id: 8,
   stream_id: 6,
   start_time: Thu, 12 May 2022 20:16:00.086639000 UTC +00:00,
   end_time: Thu, 12 May 2022 20:19:18.976072000 UTC +00:00>],
 [#<DefaultStreamHistory:0x00007f1453abf1f8
   id: 5,
   stream_id: 5,
   start_time: Wed, 11 May 2022 11:53:16.471164000 UTC +00:00,
   end_time: Wed, 11 May 2022 19:06:41.967913000 UTC +00:00>,
  #<DefaultStreamHistory:0x00007f1453abf018
   id: 7,
   stream_id: 5,
   start_time: Thu, 12 May 2022 20:15:20.620490000 UTC +00:00,
   end_time: Thu, 12 May 2022 20:16:00.068557000 UTC +00:00>,
  #<DefaultStreamHistory:0x00007f1453abee10
   id: 9,
   stream_id: 5,
   start_time: Thu, 12 May 2022 20:19:18.994331000 UTC +00:00,
   end_time: nil>]]
```

I'd expect the current default (stream 5) to show three time ranges, but it only shows the start time for the current default history. The previous history start and end times are not showing. 
<img width="1159" alt="Screen Shot 2022-05-12 at 4 23 02 PM" src="https://user-images.githubusercontent.com/458247/168161606-b9cae083-e79c-4ee6-b11f-9d848cd4e8ab.png">


It is correctly showing the start and end times for stream 6.
<img width="1176" alt="Screen Shot 2022-05-12 at 4 23 07 PM" src="https://user-images.githubusercontent.com/458247/168161632-dd8a9ebf-a6ea-4c0c-938a-2723033615d4.png">